pilot/cmd/istioctl/inject.go

```diff
@@ -108,6 +144,10 @@ istioctl kube-inject -f deployment.yaml -o deployment-injected.yaml
 
 # Update an existing deployment.
 kubectl get deployment -o yaml | istioctl kube-inject -f - | kubectl apply -f -
+
+# Create a persistent version of the deployment with Envoy sidecar
+# injected configuration from kubernetes configmap 'istio-inject'
+istioctl kube-inject -f deployment.yaml -o deployment-injected.yaml -c kubeconfig --injectConfigMapName istio-inject
```

Comment:
You can probably omit `-c kubeconfig` here. If you do want to be explicit we ought to include `--meshConfigMapName` and `--istioNamespace` as well.

Reply:
I removed the `-c kubeconfig` to keep the example concise